collect_app/src/main/java/org/odk/collect/android/formentry/saving/SaveFormProgressDialogFragment.java

```diff
@@ -36,7 +36,6 @@ public void onAttach(@NonNull Context context) {
         ViewModelProvider.Factory factory = formSaveViewModelFactoryFactory.create(requireActivity(), null);
         viewModel = new ViewModelProvider(requireActivity(), factory).get(FormSaveViewModel.class);
 
-        setCancelable(false);
```

Comment:
The default is true so this might change behavior. I see from `setCancellableFalse_makesTheDialogNotCancellable` that an argument can be set on launch. Is it always? And if it was ignored before, should we perhaps just make it always cancelable and forget about the arg?

Reply:
There is a test for this in `SaveProgressDialogFragmentTest`. I think you're right that the default is `true` for `DialogFragment` but in `ProgressDialogFragment` it looks like we make it `false` [here](https://github.com/getodk/collect/blob/5d4a1dae5520d4270a2c99c87a7420602559b2d8/collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/ProgressDialogFragment.java#L110).

Reply:
Is it the case that `setupView` is always guaranteed to be called after `onAttach`? I thinkso? If so then you're right that the call on `onAttach` was always overridden.

Reply:
Yeah I think so. I think the test proves this is true for `SaveProgressDialogFragment`.

Reply:
I don't think so? Wouldn't it only prove that if it set `cancellable` to `true`? Because previously `onAttach` already set `cancellable` to `false` so a param of `false` would have no effect.